101-vm-automatic-static-ip/README.md

```diff
@@ -0,0 +1,21 @@
+# Simple deployment of a VM with automatically assigned static IP address.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-vm-automatic-static-ip%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-vm-automatic-static-ip%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+
+This template allows you to deploy a simple Linux VM using Ubuntu from the marketplace. This will deploy a VNET, Subnet, and an A1 size VM in the resource group location with a static IP that is dynamically assigned."
```

Comment:
Clarified to: .... with a dynamically assigned IP address and then convert it to static IP. 

The point of this template is to show how you can let Azure assign the IP but then switch it to static using JSON. This is easy to do from the portal but the existing JSON samples require the user to provide the IP address.